aepsych/server/server.py

```diff
@@ -22,18 +23,19 @@
 from aepsych.server.sockets import DummySocket, createSocket
 from aepsych.strategy import SequentialStrategy
 
+
 logger = utils_logging.getLogger(logging.INFO)
 
 
 def get_next_filename(folder, fname, ext):
-    n = sum(1 for f in os.listdir(folder) if os.path.isfile(os.path.join(folder, f)))
```

Comment:
There are lots of formatting changes like this throughout the pull request. Are these from µfmt? Were we not following our own formatting guidelines before?

Reply:
I originally was using PEP8, I switched to ufmt on the latest PR.